dist.ini

```diff
@@ -9,3 +9,14 @@ no_makemaker = 1
 alien_repo = http://www.mpg123.de/download
 alien_pattern_suffix = \.tar\.bz2
 alien_bins = mpg123
+
+[Prereqs / ConfigureRequires]
+Alien::Base = 0.021
```

Comment:
This is okay, but AB isn't actually needed at configure time.

Reply:
Fixed with new commit. Thank you ! :+1: